### ISSUE TITLE
Add terra-Base class to tests

### ISSUE THREAD

**Issue by Matt-Butler** [cerner/terra-core#258]
Running the nightwatch tests, we receive warnings:
```
    console.warn packages/terra-table/node_modules/terra-base/lib/baseStyles.js:10
      The html element is missing the terra-Base class.
    console.warn packages/terra-table/node_modules/terra-base/lib/baseStyles.js:16
      The html element is missing the dir attribute. For terra directionality based styles to render correctly, add dir="ltr" or dir="rtl" to the html element.
    console.warn packages/terra-table/node_modules/terra-icon/node_modules/terra-base/lib/baseStyles.js:10
      The html element is missing the terra-Base class.
    console.warn packages/terra-table/node_modules/terra-icon/node_modules/terra-base/lib/baseStyles.js:16
      The html element is missing the dir attribute. For terra directionality based styles to render correctly, add dir="ltr" or dir="rtl" to the html element.
```

To remove these warnings, we need to update `terra-Application` to `terra-Base` here: https://github.com/cerner/terra-core/blob/master/packages/terra-toolkit/tests/nightwatch/fixtures/index.html#L2

**Comment by Matt-Butler** [comment]
It looks like the jest tests are also displaying this warning. We need to update those too.

**Comment by bjankord** [comment]
Fixed in #259
